v2/internal/identity/token_credential_provider_test.go

```diff
@@ -6,8 +6,11 @@
 package identity
 
 import (
-	"github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
```

Comment:
Can't remove this `cloud` import as it's needed on line 26 (see failing test)

Reply:
Thanks! Not sure how that got removed but its back now.